### PR TITLE
ADD language support for .edge

### PR DESCRIPTION
**Pull request by eivindml**
### What does this implement/fix? Explain your changes.

Adds support for [Edge](http://edge.adonisjs.com) templating language. It works great using the default html beautifier. 

### Does this close any currently open issues?

No.

### Any other comments?

...

### Checklist

Check all those that are applicable and complete.

- [x] Merged with latest `master` branch
- [X] Regenerate documentation with `npm run docs`
- [X] Add change details to `CHANGELOG.md` under "Next" section
- [ ] Added examples for testing to [examples/ directory](examples/)
- [ ] Travis CI passes (Mac support)
- [ ] AppVeyor passes (Windows support)

### CODE REVIEW

**Comment by Glavin001** [comment]
The [Edge website](http://edge.adonisjs.com/) is down:

image

**Comment by eivindml** [comment]
The website is back up now

File: package.json [hunk 1]
```diff
@@ -277,7 +277,8 @@
       "atom-beautify:beautify-language-xml",
       "atom-beautify:beautify-language-xtemplate",
       "atom-beautify:beautify-language-yaml",
-      "atom-beautify:beautify-language-terraform"
+      "atom-beautify:beautify-language-terraform",
+      "atom-beautify:beautify-language-edge"
```
Comment (iamkhaya) [inline package.json]: These look like they are in alphabetical order, meaning edge should be further up.

**Comment by szeck87** [comment]
@eivindml can you merge with the latest master and push changes?

File: package.json (hunk 1 above)
Comment (eivindml) [reply package.json]: Did a proper sorting using vims sorting feature.

**Comment by eivindml** [comment]
@szeck87 I merged to current branch now. Let me know if anything more should be fixed.

I also sorted some lists that where alphabetical, but not properly sorted, using vim's sort feature. So its easier to scan the lists visually when adding new stuff.

**Comment by szeck87** [comment]
Thanks @eivindml. This PR doesn't include any beautifiers for edge to use.  How will this be used exactly?

**Comment by eivindml** [comment]
@szeck87 I don't quite remember anymore. It's been such a long time since I opened the pull request, and I don't use Atom anymore. 

But it did work in the first pull request I did. Maybe something changed when I tried to update it last time? 

.edge is similar to Twig, so I think I looked at how Twig was defined in the project, and copied from there.

**Comment by Glavin001** [comment]
> Can close this issue, but there might be others interested in .edge support.

If someone else is interested in adding `.edge` support please comment and we can reopen. Thanks!

**Pull request closed without merging**